javav2/usecases/creating_dynamodb_web_app/src/main/java/com/example/services/DynamoDBService.java

```diff
@@ -1,533 +1,542 @@
-        Calendar cal = Calendar.getInstance();
-        SimpleDateFormat sdf = new SimpleDateFormat(DATE_FORMAT_NOW);
-        return sdf.format(cal.getTime());
-    }
-
-    // Convert Work data into an XML schema to pass back to client
-    private Document toXmlItem(String id2, String desc2, String status2) {
-
-        try {
-            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-            DocumentBuilder builder = factory.newDocumentBuilder();
-            Document doc = builder.newDocument();
-
-            //Start building the XML
-            Element root = doc.createElement( "Items" );
-            doc.appendChild( root );
-
-            Element item = doc.createElement( "Item" );
-            root.appendChild( item );
-
-            //Set Id
-            Element id = doc.createElement( "Id" );
-            id.appendChild( doc.createTextNode(id2 ) );
-            item.appendChild( id );
-
-            //Set Description
-            Element desc = doc.createElement( "Description" );
-            desc.appendChild( doc.createTextNode(desc2 ) );
-            item.appendChild( desc );
-
-            //Set Status
-            Element status = doc.createElement( "Status" );
-            status.appendChild( doc.createTextNode(status2 ) );
-            item.appendChild( status );
-
-            return doc;
-
-        } catch(ParserConfigurationException e) {
-            e.printStackTrace();
-        }
-        return null;
-    }
-}
-
-
-
-
-
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.services;
+
+import com.example.entities.WorkItem;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import software.amazon.awssdk.enhanced.dynamodb.*;
+import software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider;
+import software.amazon.awssdk.enhanced.dynamodb.model.UpdateItemEnhancedRequest;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.*;
+import software.amazon.awssdk.enhanced.dynamodb.model.ScanEnhancedRequest;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.StringWriter;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.*;
+import org.springframework.stereotype.Component;
+
+/*
+ Before running this code example, create a table named Work with a PK named id.

```

Comment:
Slightly confused by this - is should the table be name 'Work with a PK named id'? Or...

Reply:
Changed to to. 
/*
 Before running this code example, create an Amazon DynamoDB table named Work with a primary key named id.
 */